pkg/kubelet/allocation/allocation_manager_test.go

```diff
@@ -548,11 +648,90 @@ func TestRetryPendingResizes(t *testing.T) {
 			},
 			expectPodSyncTriggered: "true",
 		},
+		{
+			name:                         "pod-level: Request CPU and memory increase - expect InProgress",
```

Comment:
Thanks for adding these cases. They look good. I would like to request one or two cases with both pod level and container level resources